packages/core/tests/index.test.ts

```diff
@@ -1416,9 +1416,7 @@ describe('createCss: mixed(SSR & Client)', () => {
       [mediaString]: { color: 'red' },
     }).toString();
     // tslint:disable-next-line
-    expect(console.warn).toHaveBeenCalledWith(
-      `The property "color" with media query ${mediaString} can cause a specificity issue. You should create a breakpoint`
-    );
+    expect(console.warn).toHaveBeenCalled();
```

Comment:
I think we should check the message here just so that this doesn't catch some other random warning by a mistake

Reply:
Done!